### PR TITLE
Wrong capitalisation on `process.env`

### PR DESCRIPTION
**Pull request by jake-87**

### CODE REVIEW

**Comment by jake-87** [comment]
Apologies that these slipped through, as far as I can tell, they were misspelled while doing final bugfixes in the Github online editor, hence why they were not caught. I am working on unit tests for these systems, but it is more complicated due to the interactive nature of the system.

File: packages/ctl/ctl.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ async function configToKeyVal(data) {
 
 async function serializedConfigToString(serlializedConfig, configGroup, disallowedList) {
 	let allConfigElements = "";
-	for (let group of serializedConfig.groups) {
+	for (let group of serlializedConfig.groups) {
```
Comment (Danielv123) [inline packages/ctl/ctl.js:73]: Why did you add a new typo?

**Review by Danielv123 — CHANGES_REQUESTED** [review]

**Review by Danielv123 — APPROVED** [review]
Should be squash-merged

**Review by psihius — APPROVED** [review]
And this is why I run an IDE with a spell checker :100:

**Pull request merged**